docs/transparent-validation/msi-z790/laboratory-assembly-guide.md

```diff
@@ -0,0 +1,389 @@
+# Laboratory stand dedicated to MSI PRO Z690-A and Z790-P assembly guide
+
+## Introduction
+
+This document describes the assembly procedure dedicated to the MSI PRO Z690-A /
+Z790-P
```

Comment:
```suggestion
This document describes the assembly procedure dedicated to the MSI PRO Z790-P
```

Since this is Z790 directory, why do we mix Z690? Please change it everywhere (except images, these would be the same anyways)

Reply:
I still see Z690 here

Reply:
Corrected